python/langsmith/evaluation/_arunner.py

```diff
@@ -839,15 +900,8 @@ async def _arun_evaluators(
                         f" run {run.id}: {repr(e)}",
                         exc_info=True,
                     )
-                    logger.error(
-                        f"Error running evaluator {repr(evaluator)} on"
-                        f" run {run.id}: {repr(e)}",
-                        exc_info=True,
-                    )
-                if example.attachments is not None:
-                    for attachment in example.attachments:
-                        reader = example.attachments[attachment]["reader"]
-                        reader.seek(0)
```

Comment:
This no longer works because the evaluators run in parallel. I do not know the ideal solution to this, am open to any and all ideas.